src/current/v23.2/physical-cluster-replication-technical-overview.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Physical Cluster Replication Technical Overview
+summary: Understand the technical details of CockroachDB physical cluster replication.
+toc: true
+docs_area: manage
+---
+
+{{site.data.alerts.callout_info}}
+{% include feature-phases/preview.md %}
+{{site.data.alerts.end}}
+
+{% include_cached new-in.html version="v23.2" %} Physical cluster replication automatically and continuously streams data from an active _primary_ CockroachDB cluster to a passive _standby_ cluster. Each cluster contains: a _system interface_ and an application _virtual cluster_:
+
+{% include {{ page.version.version }}/physical-replication/interface-virtual-cluster.md %}
+
+This separation of concerns means that the replication stream can operate without affecting work happening in the virtual cluster.
+
+### Replication stream start-up sequence
+
+Starting a physical replication stream consists of two jobs: one on the primary and one on the standby.
+
+- Standby consumer job: Communicates with the primary cluster via an ordinary SQL connection and is responsible for initiating the replication stream. The consumer job ingests updates from the primary cluster producer job.
+- Primary producer job: Protects data on the primary cluster and sends updates to the standby cluster.
+
+The stream initialization proceeds as follows:
+
+1. The standby's consumer job connects via its system interface to the primary cluster and starts the primary cluster's physical stream producer job.
+1. The primary cluster chooses a timestamp at which to start the physical replication stream. Data on the primary is protected from [garbage collection]({% link {{ page.version.version }}/architecture/storage-layer.md %}#garbage-collection) until it is replicated to the standby using a [protected timestamp]({% link {{ page.version.version }}/architecture/storage-layer.md %}#protected-timestamps).
+1. The primary cluster returns the timestamp and a [job ID]({% link {{ page.version.version }}/show-jobs.md %}#response) for the replication job.
+1. The standby cluster retrieves a list of all nodes in the primary cluster. It uses this list to distribute work across all nodes in the standby cluster.
```

Comment:
Does it consider only active / available nodes in the primary cluster? What if there are decommissioning nodes or nodes that just happen to be down when replication is initiated?

Reply:
So, I checked in with eng. Yes, "it considers nodes it thinks are alive at the point of planning. If it turns out one of those nodes isn't alive or it dies that will result in an error, we will retry and get a new list of nodes".... I have left this as it is for now since a retry is in place. I think that if we get questions around this we may want to expand the detailed section at the bottom of the page rather than here. (I'm also going to open an issue for some follow-up and I will add this as a note to consider.)